deployment/arm/deploy-global.sh

```diff
@@ -1,23 +1,36 @@
 #!/bin/bash
+set -eu
+
 parent_path=$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P )
 cd "$parent_path"
 
 businessUnit=$1
 appName=$2
 env=$3
+
+if [[ "$#" -ne 6 ]]; then
```

Comment:
Why 6? We need at least 4 and it can go up to whatever number of regions we want to deploy to. I think it would make more sense to have a check for at least 4 vs exactly 6.

Reply:
Agree.